src/elements/pages/dev/game/pages/game-tokens.ts

```diff
@@ -13,14 +14,26 @@ import { TraversableErrors, VALIDATION_ERRORS } from '../../../../../utils/trave
 import timing, { Debounce } from '../../../../../utils/timing';
 import { map } from 'lit/directives/map.js';
 
-const PORT_PROTOCOLS: DropDownSelection<cloud.ProxyProtocol>[] = [
+const PORT_PROTOCOLS: DropDownSelection<Rivet.cloud.version.matchmaker.PortProtocol>[] = [
 	{
 		label: 'HTTP',
-		value: cloud.ProxyProtocol.HTTP
+		value: Rivet.cloud.version.matchmaker.PortProtocol.Http
 	},
 	{
 		label: 'HTTPS',
-		value: cloud.ProxyProtocol.HTTPS
+		value: Rivet.cloud.version.matchmaker.PortProtocol.Https
+	},
+	{
+		label: 'TCP',
+		value: Rivet.cloud.version.matchmaker.PortProtocol.Tcp
+	},
+	{
+		label: 'TCP TLS',
+		value: Rivet.cloud.version.matchmaker.PortProtocol.TcpTls
+	},
+	{
+		label: 'UDP',
+		value: Rivet.cloud.version.matchmaker.PortProtocol.Udp
```

Comment:
I've added it in #26, this could lead to a possible conflict.

Reply:
That's ok, we can fix the merge conflict when it arises